draft-ietf-quic-qcram.md

```diff
@@ -362,7 +362,7 @@ the number of concurrently blocked streams.
 
 ### Fixed overhead.
 
-HPACK defines overhead as 32 bytes ({{!RFC7541}} Section 4.1).  As described
+HPACK defines overhead as 32 bytes ({{!RFC7541}}, Section 4.1).  As described
```

Comment:
Is there a consistent format we should use for references to RFC7541?  Or it's ok that one is 'see Section X.Y of' and the other is ', Section X.Y'?

Reply:
I have been using "Section X.Y of {{CITATION}}" always.  The other acceptable form is "{{CITATION}}, Section X.Y", but there is value in consistency.

Reply:
...depends on context. The first form works a bit better in prose, the second one in parentheses. IMHO.